packages/migrate/src/interfaces/fix-transform.ts

```diff
@@ -1,17 +1,27 @@
 import ts from 'typescript';
+import RehearsalService from '../rehearsal-service';
```

Comment:
looks like this is being used only as a type. so instead just import the type eg. `import type RehearsalService from "../rehearsal-service";

Reply:
changed